### DISCUSSION TITLE
How can i change this function HAL_ADC_MspInit() in Platformio IDE?

### DISCUSSION THREAD

**Discussion by Industrialli** [stm32duino/Arduino_Core_STM32#2429] [Q&A] [answered]
I want to change the function HAL_ADC_MspInit(), but its already declared in core your library.

multiple definition of `HAL_ADC_MspInit'; .pio/build/industrialli_h723vet6/SrcWrapper/src/stm32/analog.cpp.o:analog.cpp

OBS: industrialli_h723vet6 is a variant board.

**Comment by fpistm** [comment, marked as answer]
You can't. Or you have to disable the Arduino analog api using this:
https://github.com/stm32duino/Arduino_Core_STM32/wiki/HAL-configuration#hal-module-only
